crates/net/network/src/peers/manager.rs

```diff
@@ -570,6 +570,21 @@ impl PeersManager {
         }
         Some((*best_peer.0, best_peer.1))
     }
+    //tick function to update reputation of all connected peers
+    fn tick(&mut self, now: Instant) {
+        let mut peers = self.peers.iter_mut().filter(|(_, peer)| {
+                         peer.state.is_connected() &&
+                         !peer.is_banned() &&
+                        (!self.connect_trusted_nodes_only || peer.is_trusted())
+                        })
```

Comment:
we only need to do this, so checking `is_connected` is sufficient here.

Reply:
ok